### ISSUE TITLE
AMP list src api url not executing after few(1000px) from the top of browser

### ISSUE THREAD

**Issue by satishsurve** [ampproject/amphtml#25605]
## What's the issue?
AMP List src url not executing after few(1000px) from the top of the browser. Initial viewport(top 1000px) sections amp-list src URL executing correctly but when we scroll down page (after some 1000px) not executing amp-list src URL.

![image](https://user-images.githubusercontent.com/3388571/68932539-d957ac80-07b8-11ea-935e-20c1e12747db.png)

We used multiple amp-list tags on a page, but when we resize the page, amp-list working correctly.

## What browsers are affected?
All browsers

## Which AMP version is affected?
Powered by AMP ⚡ HTML – Version 1911070201440

Thanks in advance!

**Comment by nainar** [comment]
Can you provide a demo of the code that is failing?

**Comment by cathyxz** [comment]
There’s been a similar issue filed recently about images not loading 1000 px below the initial viewport. I wonder if layoutCallback is correctly called. You can check this by setting a breakpoint in amp-lists layoutCallback with a repro. If layoutCallback is called, I suspect there should be error messages in the console that would help explain what’s up. If layoutCallback is not called, we should look into recent runtime changes that may impact how elements below initial viewport are laid out. 

But a code pen link to a repro would be nice yes. =)

**Comment by cathyxz** [comment]
Oh I see. You filed https://github.com/ampproject/amphtml/issues/25555 too. Hmm. @jridgewell would you by chance know if something’s up?

**Comment by jridgewell** [comment]
Closing as a duplicate of https://github.com/ampproject/amphtml/issues/24929, see https://github.com/ampproject/amphtml/issues/25555#issuecomment-557307916 for an explanation why it's a duplicate.
